joomla/joomla-cms
# replace float-end by float-md-end

Pull Request for Issue #40621 .

### Summary of Changes

This PR will prevent the the element with className `com-content-category-blog__counter` be shown behind the pagination. It will get float-end on breakpoint md. 

### Testing Instructions

- Joomla with blog sample data
- Make sure to copy blog content items so you have a pagination with a lot of pages 
- Reduce the viewport from desktop to mobile


### Actual result BEFORE applying this Pull Request

The counter is behind the pagination as shown in the issue #40621


### Expected result AFTER applying this Pull Request

The counter does not have a float and will be shown above the pagination


### Link to documentations
Please select:
